### ISSUE TITLE
[Bug] Update dep poem for fix

### ISSUE THREAD

**Issue by JasperTimm** [aptos-labs/aptos-core#9358]
# 🐛 Bug

The dependency poem is currently at 1.3.55 on the devnet branch. It has a compatibility error with the tracing dependency, see here: [](https://github.com/poem-web/poem/pull/578/files)

## To reproduce

Attempt to include the aptos-sdk from the devnet branch in a Rust crate and compile

**Stack trace/error message**
```
   Checking poem v1.3.55 (https://github.com/poem-web/poem.git?branch=master#9941b489)
error[E0277]: the size for values of type `str` cannot be known at compilation time
```

## Expected Behavior

Compiles successfully

## System information

**Please complete the following information:**
-  devnet branch
-  1.70.0
-  Linux

## Additional context

Should be easily fixable by updating the poem dep to the latest version (0.3.57)

**Comment by banool** [comment]
Updating Poem to the latest version is a breaking change unfortunately. I have looked into upgrading the Poem version in the past but it has been quite difficult to get it to work due to some of the macros we have.

**Comment by JasperTimm** [comment]
Fair enough. I'm not an expert Rustacean so I don't know the specifics of dependency handling, but is it possible for you to put in the same change that the Poem folks did, i.e. pin `tracing = "0.1.36"`?

In any case, for anyone else that's struggling with this, that's the fix I've put in my own Cargo.toml for now.
